src/main/java/com/enonic/lib/cron/scheduler/JobExecutionCommand.java

```diff
@@ -52,7 +52,10 @@ public void run()
             else
             {
                 this.doRun();
-                this.rerunCallback.accept( this );
+                if ( this.descriptor.getFixedDelay() == 0 )
```

Comment:
rerunCallback could be set as noop instead when JobExecutionCommand is constructed